pythainlp/cli/data.py

```diff
@@ -0,0 +1,86 @@
+"""
+thainlp dataset/corpus management command line.
+"""
+import argparse
+
+from pythainlp import cli, corpus
+from pythainlp.tools import get_pythainlp_data_path
+
+
+class App:
+    def __init__(self, argv):
+        parser = argparse.ArgumentParser(
+            prog="data", usage="thainlp data <subcommand>",
+        )
+        parser.add_argument(
+            "subcommand",
+            type=str,
+            # choices=["catalog", "info", "get", "rm", "path", "ls"],
+            help="action on dataset/corpus (catalog, info, get, rm, path)",
+        )
+
+        args = parser.parse_args(argv[2:3])
+        if not hasattr(App, args.subcommand):
+            print(f"Subcommand not available: {args.subcommand}")
+            parser.print_help()
+            exit(1)
```

Comment:
what do you if we raise an exception here instead?

For example, we might use `raise ValueError(...)`.

Ref: https://docs.python.org/3/library/exceptions.html#ValueError

Reply:
Decided to go with argparser's built-in `choices=[..]`. Let them handle it.